libs/platform/experience/src/services/layout/plugins/properties/divider/divider.styles.ts

```diff
@@ -29,9 +25,13 @@ export const styles: LayoutStyles = {
         )
       );
     }
+  `,
+};
 
-    :host([layout-vertical]) *:not(:first-child)::before,
-    :host([layout-vertical]) ::slotted(*:not(:first-child))::before {
+export const verticalStyles: LayoutStyles = {
```

Comment:
```suggestion
export const verticalDividerStyles: LayoutStyles = {
```

Or `dividerStyles.vertical`